pkg/collect/helm.go

```diff
@@ -45,6 +49,32 @@ type VersionInfo struct {
 	Values    map[string]interface{} `json:"values,omitempty"`
 }
 
+type configGetter struct {
+	restConfig *rest.Config
+}
+
+// ToDiscoveryClient implements genericclioptions.RESTClientGetter.
+func (c configGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
+	return nil, nil
+}
+
+// ToRESTConfig implements genericclioptions.RESTClientGetter.
+func (c configGetter) ToRESTConfig() (*rest.Config, error) {
+	return c.restConfig, nil
+}
+
+// ToRESTMapper implements genericclioptions.RESTClientGetter.
+func (c configGetter) ToRESTMapper() (meta.RESTMapper, error) {
+	return nil, nil
+}
+
+// ToRawKubeConfigLoader implements genericclioptions.RESTClientGetter.
+func (c configGetter) ToRawKubeConfigLoader() clientcmd.ClientConfig {
+	return nil
+}
```

Comment:
I see this is possible to get the clientcmd.ClientConfig with k8sutil. It may be best to pass this via args similar to ClientConfig via the opts structs ([here](https://github.com/replicatedhq/troubleshoot/blob/main/cmd/collect/cli/run.go#L160), [here](https://github.com/replicatedhq/troubleshoot/blob/main/cmd/troubleshoot/cli/run.go#L180), [here](https://github.com/replicatedhq/troubleshoot/blob/main/pkg/preflight/run.go#L355), and [here](https://github.com/replicatedhq/troubleshoot/blob/main/pkg/preflight/run.go#L410)) although I do see collectors using the function [directly](https://github.com/replicatedhq/troubleshoot/blob/main/pkg/collect/copy_from_host.go#L73).

```suggestion
func (c configGetter) ToRawKubeConfigLoader() clientcmd.ClientConfig {
	return k8sutil.GetKubeconfig()
}
```